src/main/java/com/linkedin/kmf/services/ConsumeService.java

```diff
@@ -63,8 +63,8 @@
 
   public ConsumeService(Map<String, Object> props, String name) throws Exception {
     _name = name;
-    Map consumerPropsOverride = props.containsKey(ConsumeServiceConfig.CONSUMER_PROPS_CONFIG) ?
-        (Map) props.get(ConsumeServiceConfig.CONSUMER_PROPS_CONFIG) : new HashMap<>();
+    Map consumerPropsOverride = props.containsKey(ConsumeServiceConfig.CONSUMER_PROPS_CONFIG)
+      ? (Map) props.get(ConsumeServiceConfig.CONSUMER_PROPS_CONFIG) : new HashMap<>();
```

Comment:
The existing code style seems better to me. The idea is that when we read `?` at the end of first line, it is more obvious that something else will follow. Can we keep the existing style?